storage/store/inmemory/unsynchronized/collections.go

```diff
@@ -128,3 +128,27 @@ func (c *Collections) Remove(collID flow.Identifier) error {
 
 	return nil
 }
+
+// Collections returns a copy of stored collections
+func (c *Collections) Collections() []flow.Collection {
```

Comment:
since all of the other methods use the function `Data()`, I think we should do the same here. You could just have it return both data types